1molehayo/portfolio
# #Refactor: update summitech  project

### What does this MR do?
- update summitech project image 
- update readme docs
- fix image scaling bug

### How can it be manually tested?
- launch website 
- navigate to works to see changes
